client/src/pages/HomePage.js

```diff
@@ -1,28 +1,68 @@
 import React from "react";
 import AppBarLoggedIn from "../layout/AppBarLoggedIn";
 import Mention from "../layout/Mention";
-import { Grid } from "@material-ui/core";
+import MentionList from "../layout/MentionList";
+import { Grid, Typography, Box } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
 
-const HomePage = ({ match }) => {
+const useStyles = makeStyles((theme) => ({
+  box: {
+    alignItems: "center",
+    margin: theme.spacing(6, 3, 4, 0),
+    width: "80%",
+  },
+}));
+
+const HomePage = () => {
+  const classes = useStyles();
   const mentionDatas = [
     {
-      title: "PayPal invested $400 in DolphinCorp",
+      title: "PayPal invested $400 in DolphinCorp PayPal invested $400",
       id: 1,
       platform: "Reddit",
-      image: "https://upload.wikimedia.org/wikipedia/fr/f/fc/Reddit-alien.png",
+      image: "https://picsum.photos/100",
+      content:
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
+    },
+    {
+      title: "PayPal invested $800 in DolphinCorp",
+      id: 2,
+      platform: "Facebook",
+      image: "https://picsum.photos/200/300",
+      content:
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
+    },
+    {
+      title: "invested $400 in DolphinCorp",
+      id: 3,
+      platform: "Twitter",
+      image: "https://picsum.photos/200",
+      content:
+        "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
+    },
+    {
+      title: "invested $400 in DolphinCorp",
+      id: 4,
+      platform: "Twitter",
+      image: "https://picsum.photos/400",
       content:
         "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisquis nostrud exercitation ullamco laboris nisUt eni",
     },
   ];
 
-  const k = match.params;
   return (
     <>
       <AppBarLoggedIn />
-      {console.log(k)}
       <Grid container style={{ backgroundColor: "#fff", height: "100vh" }}>
```

Comment:
Preferred here to avoid styling in multiple locations: move these inline styles to a class in useStyles

Reply:
sure, I will move these inline styles to makestyle after.